pkg/controller/user/user.go

```diff
@@ -51,17 +51,28 @@ func (lc *userListController) ServeHTTP(w http.ResponseWriter, r *http.Request)
 	}
 	flash := flash.FromContext(w, r)
 
-	lc.logger.Infof("FLASH: %+v", flash)
+	realm, err := controller.GetRealm(w, r, user)
+	if err != nil {
+		flash.Error("Select realm to continue.")
+		http.Redirect(w, r, "/realm", http.StatusSeeOther)
+		return
+	}
 
 	m := html.GetTemplateMap(r)
 	m["user"] = user
+	m["realm"] = realm
 
-	users, err := lc.db.ListUsers(false)
-	if err != nil {
+	if err := realm.LoadRealmUsers(lc.db, false); err != nil {
 		flash.ErrorNow("Error loading users: %v", err)
 	}
 
-	m["users"] = users
+	admins := make(map[uint]bool)
+	for _, au := range realm.RealmAdmins {
```

Comment:
Is RealmAdmins populated by LoadRealmUsers? That's not super intuitive to me... I'd rather see this as `RealmAdmins()` that's loaded and cached.